llvm/lib/Target/AArch64/AArch64Processors.td

```diff
@@ -983,50 +1007,58 @@ def : ProcessorModel<"thunderx3t110", ThunderX3T110Model,
 def : ProcessorModel<"tsv110", TSV110Model, ProcessorFeatures.TSV110,
                      [TuneTSV110]>;
 
+// iPhone, iPad, Mac and Watch CPUs
```

Comment:
Certainly minor, but maybe just "Apple CPUs"

Reply:
works for me